### PR TITLE
README was created

### PR DESCRIPTION
**Pull request by farfardesu**

### CODE REVIEW

**Review by SarumanIsAlive — APPROVED** [review]

**Comment by Nick97-git** [comment]
Comments about Readme:
- Technologies that was used -> Technologies
- Add more technologies and separate JSTL and JSP
- How to run a project -> make a numbered list of it and write it like successive steps
- Input schema from "main/resources/init_db.sql" to MySQL workbench to configure DB -> Use "main/resources/init_db.sql" in MySQL workbench to configure schema
- Functionality that is implemented -> Implemented functionality
- (DI in future) can be removed
- is to efficiently -> remove to
Comments about code:
- remove /all from /cars/all, /drivers/all, /manufacturers/all url patterns
- init_db.sql -> add columns password and login to drivers table
- move GetMyCurrentCarsController to `car` package
- LoginController -> Driver user -> Driver driver
- AuthenticationServiceImpl -> remove empyt line before @Inject
- String USER_ID = "user_id" -> String DRIVER_ID = "driver_id" change everywhere where it's used
- AuthenticationFilter -> Long userId -> Long driverId

File: src/main/java/taxi/controller/car/AddCarController.java [hunk 1]
```diff
@@ -12,6 +12,7 @@
 import taxi.service.ManufacturerService;
 
 public class AddCarController extends HttpServlet {
+    private static final String MANUFACTURER_ID = "manufacturer_id";
```
Comment (Nick97-git) [inline src/main/java/taxi/controller/car/AddCarController.java]: don't need to create constant for this

File: src/main/java/taxi/controller/car/AddDriverToCarController.java [hunk 2]
```diff
@@ -12,6 +12,8 @@
 import taxi.service.DriverService;
 
 public class AddDriverToCarController extends HttpServlet {
+    private static final String CAR_ID = "car_id";
+    private static final String DRIVER_ID = "driver_id";
```
Comment (Nick97-git) [inline src/main/java/taxi/controller/car/AddDriverToCarController.java]: the same

File: src/main/java/taxi/controller/car/GetMyCurrentCarsController.java [hunk 3]
```diff
@@ -12,6 +12,7 @@
 import taxi.service.CarService;
 
 public class GetMyCurrentCarsController extends HttpServlet {
+    private static final String USER_ID = "user_id";
```
Comment (Nick97-git) [inline src/main/java/taxi/controller/car/GetMyCurrentCarsController.java]:
```suggestion
    private static final String DRIVER_ID = "driver_id";
```

File: src/main/java/taxi/controller/car/GetMyCurrentCarsController.java [hunk 4]
```diff
@@ -20,7 +21,7 @@
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
         HttpSession session = req.getSession();
-        Long userId = (Long) session.getAttribute("user_id");
+        Long userId = (Long) session.getAttribute(USER_ID);
```
Comment (Nick97-git) [inline src/main/java/taxi/controller/car/GetMyCurrentCarsController.java]:
```suggestion
        Long driverId = (Long) session.getAttribute(DRIVER_ID);
```

File: src/main/java/taxi/util/ConnectionUtil.java [hunk 5]
```diff
@@ -6,10 +6,11 @@
...
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL =
+            "jdbc:mysql://localhost:3306/taxi?useUnicode=true&serverTimezone=UTC";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "bloodystorm";
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
```
Comment (Nick97-git) [inline src/main/java/taxi/util/ConnectionUtil.java]: return the previous version

File: src/main/webapp/web.xml [hunk 6]
```diff
@@ -86,7 +86,7 @@
     </servlet>
     <servlet-mapping>
         <servlet-name>allManufacturers</servlet-name>
-        <url-pattern>/manufacturers/all</url-pattern>
```
Comment (Nick97-git) [inline src/main/webapp/web.xml:89]: you forgot to remove these /all from DeleteCarController, DeleteDriverController and DeleteManufacturerController

**Review by Nick97-git — CHANGES_REQUESTED** [review]

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**